### PR TITLE
Use dwi run-01 if multiple dwi runs exist

### PR DESCRIPTION
**Pull request by renelabounek**
PR is related to the issue #254

### CODE REVIEW

File: process_data.sh [hunk 1]
```diff
@@ -250,10 +250,18 @@ sct_process_segmentation -i ${file_t2s_seg}.nii.gz -angle-corr 0 -vert 3:4 -vert
...
+  file_dwi="${SUBJECT}_dwi"
+else
+  file_dwi="${SUBJECT}_run-01_dwi"
+fi
 # If there is an additional b=0 scan, add it to the main DWI data
-concatenate_b0_and_dwi "${SUBJECT}_acq-b0_dwi" $file_dwi
+if [ -f ${SUBJECT}_acq-b0_dwi.nii.gz ];then
+  concatenate_b0_and_dwi "${SUBJECT}_acq-b0_dwi" $file_dwi
+else
+  concatenate_b0_and_dwi "${SUBJECT}_acq-b0_run-01_dwi" $file_dwi # RL-20211208: I am not 100% sure, if this is the right naming format for $1
```
Comment (jcohenadad) [inline process_data.sh:263]: What do you mean by '$1'?

**Review by jcohenadad — APPROVED** [review]
looks like a good workaround, thank you @renelabounek !

File: process_data.sh (hunk 1 above)
Comment (renelabounek) [reply process_data.sh:263]: I meant: "${SUBJECT}_acq-b0_run-01_dwi" If it is accurate or if e.q. the right strin is: "${SUBJECT}_run-01_acq-b0_dwi"

Comment (jcohenadad) [reply process_data.sh:263]: it should be `"${SUBJECT}_run-01_acq-b0_dwi"`

Comment (renelabounek) [reply process_data.sh]: Repaired in commit b3075a7

**Pull request merged**